File: app/database/models/Lanzadera.js
module.exports = (sequelize, dataTypes) => {
    let alias = 'Lanzadera';
    let cols = {
        id: {
            type: dataTypes.INTEGER.UNSIGNED,
            primaryKey: true,
            autoIncrement: true,
            allowNull: false
        },
        name: {
            type: dataTypes.STRING(50),
            allowNull: false
        },
        country: {
            type: dataTypes.STRING(50)
        },
        activity_start: {
            type: dataTypes.INTEGER
        },
        activity_end: {
            type: dataTypes.INTEGER
        },
        fuel: {
            type: dataTypes.STRING(50)
        },
        cargo: {
            type: dataTypes.STRING(50)
        }
    };
    let config = {
        tableName: 'naves_lanzaderas',
        timestamps: false
    }
    const Lanzadera = sequelize.define(alias, cols, config); 
 
    return Lanzadera
};

File: spaceStation/src/main/java/challenge/spacestation/data/modelos/Lanzadera.java
package challenge.spacestation.data.modelos;

/**
 *
 * @author mcami
 */
public class Lanzadera extends Nave{
    public String cargo;

    public Lanzadera() {
    }

    public Lanzadera(String cargo, int id, String name, String country, int activity_start, int activity_end, String fuel) {
        super(id, name, country, activity_start, activity_end, fuel);
        this.cargo = cargo;
    }

    public String getCargo() {
        return cargo;
    }

    public void setCargo(String cargo) {
        this.cargo = cargo;
    }
    
    
}
